blazor/datagrid/filtering.md

```diff
@@ -7,108 +7,422 @@ control: DataGrid
+        {
+            this.OrderID = OrderID;    
+            this.CustomerID = CustomerID;
+            this.ShipCity = ShipCity;
+            this.ShipName = ShipName;           
+        }
+ 
+        public static List<OrderData> GetAllRecords()
+        {
+            if (Orders.Count() == 0)
+            {
+                int OrderID = 10248;
+ 
+                for (int i = 1; i < 3; i++)
+                {
+                    Orders.Add(new OrderData(OrderID+1, "VINET", "Reims", "Vins et alcools Chevali"));
+                    Orders.Add(new OrderData(OrderID+2, "TOMSP", "Münster", "Toms Spezialitäten"));
+                    Orders.Add(new OrderData(OrderID+3, "HANAR", "Rio de Janeiro", "Hanari Carnes"));
+                    Orders.Add(new OrderData(OrderID+4, "VICTE", "Lyon", "Victuailles en stock"));
+                    Orders.Add(new OrderData(OrderID+5, "SUPRD", "Charleroi", "Suprêmes délices"));
+                    Orders.Add(new OrderData(OrderID+6, "HANAR", "Lyon", "Hanari Carnes"));
+                    Orders.Add(new OrderData(OrderID+7, "CHOPS", "Rio de Janeiro", "Chop-suey Chinese"));
+                    Orders.Add(new OrderData(OrderID+8, "RICSU", "Münster", "Richter Supermarkt"));
+                    Orders.Add(new OrderData(OrderID+9, "WELLI", "Reims", "Wellington Import"));
+                    OrderID += 9;
+                }
+            }
+            return Orders;
+        }
+ 
         public int? OrderID { get; set; }
         public string CustomerID { get; set; }
-        public DateTime? OrderDate { get; set; }
-        public double? Freight { get; set; }
+        public string ShipCity { get; set; }
+        public string ShipName { get; set; }
+   }
+{% endhighlight %}
+{% endtabs %}
+
+{% previewsample "https://blazorplayground.syncfusion.com/embed/rXhTjMilotUqJJMF?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
+
+### Initial filter with multiple values for different columns
+
+In the Syncfusion Blazor Grid, you can establish an initial filter containing multiple values for a different column, which helps you to preset filter conditions for a different column using multiple values. This functionality allows you to display a filtered records in the grid right after the grid is initially loaded.
+
+To apply the filter with multiple values for different column at initial rendering by using Grid [Query](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_Query) property 
+
+The following example demonstrates how to perform an initial filter with multiple values for different **Order ID** and **Customer ID** columns using `Query` property
+
+{% tabs %}
+{% highlight razor tabtitle="Index.razor" %}
+@using Syncfusion.Blazor.Grids
+@using Syncfusion.Blazor.Data;
+
+<SfGrid DataSource="@GridData" @ref="Grid" Query="@Qry" AllowFiltering="true" Height="273px">
```

Comment:
Query should not be used. intead use GridFilterColumn tag

Reply:
solved